Dockerfile

```
@@ -3,15 +3,15 @@
 
 # Should be the specific version of `node:slim`.
 # `sqitch` requires at least `buster`.
-FROM node:19.7.0-slim AS development
+FROM node:19.7.0 AS development
 
 COPY ./docker/entrypoint.sh /usr/local/bin/
 
 # Update and install dependencies.
 # - `libdbd-pg-perl postgresql-client sqitch` is required by the entrypoint
 RUN apt-get update \
     && apt-get install --no-install-recommends -y \
-        libdbd-pg-perl postgresql-client sqitch \
+    libdbd-pg-perl postgresql-client sqitch \
```

Let's keep this indentation:
```suggestion
       libdbd-pg-perl postgresql-client sqitch \
```